README.md

```diff
@@ -775,11 +778,127 @@ routes => [
 #####`type`
 *Optional* The type of VPN gateway. The only currently supported value --- and the default --- is 'ipsec.1'.
 
-#### Type: route53*
+#### Type: rds_db_parameter_group
+
+Note that this type is currently only available to list via puppet
```

Comment:
I am not totally sure what this means. I think it is something like: This type can only be listed via `puppet resource